app/common/services/translation.service.js

```diff
@@ -42,20 +42,31 @@ function (
         });
     };
 
-    var getLanguage = function () {
+    var getLanguage = function (config) {
```

Comment:
It was there so I could call get language within the config endpoint. Only happened within some custom Kuwait code.